docs/Actions.md

```diff
@@ -592,6 +592,41 @@ set_changelog(app_identifier: "com.krausefx.app", version: "1.0", changelog: "Al
 
 You can store the changelog in `./fastlane/changelog.txt` and it will automatically get loaded from there. This integration is useful if you support e.g. 10 languages and want to use the same "What's new"-text for all languages.
 
+### upload_artifact
+
+This allows you to upload your ipa, or any other file you want, to artifactory. I recommend using a combination of the ENV variables along with passing some parameters into the command, as seen below.
+
+```ruby
+# This example shows the minimum required configuration to upload an artifact 
+ENV['FL_UPLOAD_ARTIFACT_ENDPOINT'] = 'https://artifactory.example.com/artifactory/'
+ENV['FL_UPLOAD_ARTIFACT_USERNAME'] = 'username'
+ENV['FL_UPLOAD_ARTIFACT_PASSWORD'] = 'password'
+
+upload_artifact(
+  file: 'example.ipa',                                # File to upload
+  repo: 'mobile_artifacts',                           # Artifactory repo
+  repo_path: '/ios/appname/example-major.minor.ipa'   # Path to place the artifact including its filename
+)
+```
+
+The full set of parameters are:
+
+```ruby
```

Comment:
Please remove the list of parameters and add this instead:

To get a list of all available parameters run `fastlane action upload_artifact`